### PR TITLE
Fix lens regression

### PR DESCRIPTION
**Pull request by yyoncho**
Fixes https://github.com/emacs-lsp/lsp-metals/issues/11

### CODE REVIEW

**Comment by yyoncho** [comment]
@kurnevsky I was unable to test with metals but most likely this will fix your issue.

File: lsp-mode.el [hunk 1]
```diff
@@ -2480,7 +2480,7 @@ BUFFER-MODIFIED? determines whether the buffer is modified or not."
   (-doto (make-sparse-keymap)
     (define-key [mouse-1] (lsp--lens-create-interactive-command command))))
 
-(lsp-defun lsp--lens-create-interactive-command ((&CodeLens :command?))
+(lsp-defun lsp--lens-create-interactive-command (command?)
```
Comment (kurnevsky) [inline lsp-mode.el:2483]: Should it be just a `defun`?

Comment (yyoncho) [reply lsp-mode.el:2483]: Yes.

**Comment by kurnevsky** [comment]
Just tested - works fine.

**Pull request merged**